Makefile

```diff
@@ -40,8 +40,8 @@ BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
 # Image URL to use all building/pushing image targets
 IMG ?= $(IMAGE_TAG_BASE):latest
-# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd"
+# Create Kubernetes objects with embeddedObjectMeta
+CRD_OPTIONS ?= "crd:generateEmbeddedObjectMeta=true"
```

Comment:
Well, actually I was planning to remove the ``Meta EmbeddedObjectMeta `json:"metadata,omitempty"``` which tries to solve the same thing by copying everything (instead of generating), but it's not part of this PR so I'll remove this change.